app/components/Button/index.tsx

```diff
@@ -8,8 +8,9 @@ type ButtonProps = {
   className?: string
   tooltip?: string
   icon?: ReactNode
+  disabled?: boolean
 }
-const Button = ({children, action, tooltip, icon, className}: ButtonProps) => {
+const Button = ({children, action, tooltip, icon, className, disabled = false}: ButtonProps) => {
```

Comment:
how would this work for link buttons? You can add a `onClick={(e) => e.preventDefault()}` or something, to disable it, but it might require a bit extra styling

Reply:
is it necessary for anything else besides the feedback form?

Reply:
not now, but it would be good to have it done for consistency